<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1595622043839' stop='1595622265748'>
    <name>Paid blocks</name>
    <title>Paid blocks</title>
    <test-cases>
        <test-case start='1595622083138' status='failed' stop='1595622118493'>
            <name>MailChimp Block</name>
            <title>MailChimp Block</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1595622083139' status='undefined'>
                    <name>Can visit the block editor and add a MailChimp block</name>
                    <title>Can visit the block editor and add a MailChimp block</title>
                    <attachments>
                        <attachment title='Test failed: Paid blocks :: MailChimp Block' source='042373a5-0a34-4d99-b444-1d147ecb9251-attachment.png' type='image/png' size='44414'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log
                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. 
                
                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... 
                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... donecat: wp-content/debug.log: No such file or directory
                ✖ Command failed with exit code 1
                Command failed with exit code 1
                error Command failed with exit code 1.
                </message>
                <stack-trace>Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log
                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. 
                
                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... 
                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... donecat: wp-content/debug.log: No such file or directory
                ✖ Command failed with exit code 1
                Command failed with exit code 1
                error Command failed with exit code 1.
                
                    at checkExecSyncError (child_process.js:630:11)
                    at execSync (child_process.js:666:15)
                    at execSyncShellCommand (/home/travis/build/Automattic/jetpack/tests/e2e/lib/utils-helper.js:27:9)
                    at _callee10$ (/home/travis/build/Automattic/jetpack/tests/e2e/lib/page-helper.js:209:9)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.&lt;computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
                    at /home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7</stack-trace>
            </failure>
        </test-case>
        <test-case start='1595622118493' status='failed' stop='1595622189282'>
            <name>Pay with PayPal</name>
            <title>Pay with PayPal</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1595622118494' status='undefined'>
                    <name>Can visit the block editor and add a Pay with PayPal block</name>
                    <title>Can visit the block editor and add a Pay with PayPal block</title>
                    <attachments>
                        <attachment title='Test failed: Paid blocks :: Pay with PayPal' source='3d8df093-95c7-4a25-9212-fe8d1550ea47-attachment.png' type='image/png' size='45080'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log
                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. 
                
                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... 
                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... donecat: wp-content/debug.log: No such file or directory
                ✖ Command failed with exit code 1
                Command failed with exit code 1
                error Command failed with exit code 1.
                </message>
                <stack-trace>Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log
                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. 
                
                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... 
                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... donecat: wp-content/debug.log: No such file or directory
                ✖ Command failed with exit code 1
                Command failed with exit code 1
                error Command failed with exit code 1.
                
                    at checkExecSyncError (child_process.js:630:11)
                    at execSync (child_process.js:666:15)
                    at execSyncShellCommand (/home/travis/build/Automattic/jetpack/tests/e2e/lib/utils-helper.js:27:9)
                    at _callee10$ (/home/travis/build/Automattic/jetpack/tests/e2e/lib/page-helper.js:209:9)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.&lt;computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
                    at /home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7</stack-trace>
            </failure>
        </test-case>
        <test-case start='1595622189282' status='failed' stop='1595622260168'>
            <name>WordAds block</name>
            <title>WordAds block</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1595622189283' status='undefined'>
                    <name>Can visit the block editor and add a WordAds block</name>
                    <title>Can visit the block editor and add a WordAds block</title>
                    <attachments>
                        <attachment title='Test failed: Paid blocks :: WordAds block' source='4ee99d06-8e07-4ca1-9e05-768501924e27-attachment.png' type='image/png' size='68318'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log
                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. 
                
                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... 
                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... donecat: wp-content/debug.log: No such file or directory
                ✖ Command failed with exit code 1
                Command failed with exit code 1
                error Command failed with exit code 1.
                </message>
                <stack-trace>Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log
                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. 
                
                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... 
                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... donecat: wp-content/debug.log: No such file or directory
                ✖ Command failed with exit code 1
                Command failed with exit code 1
                error Command failed with exit code 1.
                
                    at checkExecSyncError (child_process.js:630:11)
                    at execSync (child_process.js:666:15)
                    at execSyncShellCommand (/home/travis/build/Automattic/jetpack/tests/e2e/lib/utils-helper.js:27:9)
                    at _callee10$ (/home/travis/build/Automattic/jetpack/tests/e2e/lib/page-helper.js:209:9)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.&lt;computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
                    at /home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>